### PR TITLE
Ladtableset naming clarity

### LINKED ISSUE #3254: Rename methods and variables in LadTableSet.vue for more clarity
Names like primary and secondary were a bit confusing, changing to ladTables and telemetryObjects should make things a bit more clear.

## Testing
Create a LAD Table set
Add/Remove LAD Tables, reorder, make sure everything works as expected

### PR DESCRIPTION
**Pull request by jvigliotta**
Renamed method and variable names like "primary" and "secondary" to "ladTable" and "telemetryObject" for more clarity.

Closes #3254 

## Author Checklist
Changes address original issue? Yes
Unit tests included and/or updated with changes? N/A
Command line build passes? Yes
Changes have been smoke-tested? Yes
Testing instructions included? Yes

### CODE REVIEW

File: src/plugins/LADTable/components/LadTableSet.vue [hunk 1]
```diff
@@ -124,39 +124,39 @@ export default {
...
-                let secondary = {};
-                secondary.key = this.openmct.objects.makeKeyString(domainObject.identifier);
-                secondary.domainObject = domainObject;
+                let telemetryObject = {};
+                telemetryObject.key = this.openmct.objects.makeKeyString(domainObject.identifier);
+                telemetryObject.domainObject = domainObject;
 
-                let array = this.secondaryTelemetryObjects[primary.key];
-                array.push(secondary);
+                let array = this.ladTelemetryObjects[ladTable.key];
```
Comment (nikhilmandlik) [inline src/plugins/LADTable/components/LadTableSet.vue:146]: `array` -> `telemetryObjects`?

**Review by nikhilmandlik — CHANGES_REQUESTED** [review]
small rename for consistency :)

File: src/plugins/LADTable/components/LadTableSet.vue (hunk 1 above)
Comment (jvigliotta) [reply src/plugins/LADTable/components/LadTableSet.vue:146]: yep! good call, updated

**Review by nikhilmandlik — APPROVED** [review]

**Comment by nikhilmandlik** [comment]
### Reviewer Checklist
Check | Passed
-- | --
Changes appear to address issue? | Y
Appropriate unit tests included? | NA
Code style and in-line documentation are appropriate? | Y
Commit messages meet standards? | Y
Has associated issue been labelled unverified? (only applicable if this PR closes the issue) | Y

**Pull request merged**